lib/ditcher.js

```diff
@@ -590,6 +590,9 @@ Ditcher.prototype.doImport = function (params, cb) {
     var collectionsToImportArray = Object.keys(collectionsToImport);
 
     collectionsToImportArray.forEach(function(colKey){
+      if (collectionsToImport[colKey].length === 0) {
+        return cb(new Error("collection " + colKey + " is empty"));
```

Comment:
If there is more than one collection and if any of those collections are empty then an error is returned. Should we allow users to upload empty collections? The ticket just mentioned to return a useful error?